srcpkgs/feedbackd/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'feedbackd'
+pkgname=feedbackd
+version=0.0.0+git20200707
+revision=1
+wrksrc=${pkgname}-v${version}
+build_style=meson
+configure_args="-Dvapi=false -Dintrospection=disabled"
+hostmakedepends="glib-devel pkg-config"
+makedepends="gsound-devel libgudev-devel json-glib-devel"
+depends="dbus"
+short_desc="Daemon to provide haptic feedback on events"
+maintainer="Julio Galvan <julio-void@epazote.net>"
+license="GPL-3.0-or-later"
+homepage="https://source.puri.sm/Librem5/feedbackd"
+distfiles="${homepage}/-/archive/v${version}/${pkgname}-v${version}.tar.gz"
+checksum=37cf48cbdbaee8e2d0bea406f5b6317bf0e3cdf79250f9ce0a884e7d3905a8ea
+
+libfeedback_package() {
+	short_desc+=" - shared libraries"
+	pkg_install() {
+		vmove "usr/lib/*.so.*"
+	}
+}
+libfeedback-devel_package() {
```

Comment:
It might make sense to name this sub package `feedback-devel` instead, but I'm not sure.

Reply:
Should I rename it to `feedback-libs` and `feedback-devel`?

Reply:
Honestly, given that @fosslinux suggested these specific names, I'm okay with keeping them. It was mostly an unnecessary comment.